### DISCUSSION TITLE
ReferenceError: deck is not defined

### DISCUSSION THREAD

**Discussion by nrichers** [Martinomagnifico/reveal.js-multimodal#1] [General]
@Martinomagnifico, I recently discovered your Multimodal plugin which looks very promising. I am trying to teach myself how to use it with Quarto revealjs presentations, but I run into an issue I cannot troubleshoot. 

Specifically, I am seeing this `deck is not defined` error when I have the plugin enabled:

<img width="1920" alt="image" src="https://github.com/Martinomagnifico/reveal.js-multimodal/assets/15148011/595c6b9e-e0d7-4a54-850e-0cd81dbf2552">

<br>The grey rectangle in the top left appears to be the viewport:

<img width="1702" alt="image" src="https://github.com/Martinomagnifico/reveal.js-multimodal/assets/15148011/42a9ffb2-9b5f-4ea8-84ea-4314979e2c27">

Any idea for what might be causing this, or what information would help to pinpoint the cause? Revealjs presentations otherwise work without issue under Quarto.

**Comment by Martinomagnifico** [comment]
Hi Nik, 
Good catch. I’ll check out the “not defined” error. As for the other error, that is probably because Quarto will not autoload the CSS. I’ll create a Quarto version of this plugin anyway, may take some time.

**Comment by Martinomagnifico** [comment]
The 'not defined' error was an error where I used the deck object instead of an HTML element. That is now fixed. As for Quarto, that plugin can now be found here: [quarto-multimodal](https://github.com/martinomagnifico/quarto-multimodal). You can install the extension like this: `quarto add martinomagnifico/quarto-multimodal`.

**Comment by nrichers** [comment]
@Martinomagnifico that was quick! Thank you for fixing and for creating a Quarto extension. I'll test this as part of one of our training modules the next time we touch them.
